gameswap-service/gameswap.yml

```diff
@@ -3,20 +3,9 @@ sampleProperty: hello!
 redirectAllToHttps: false
 googleSecret: ${GOOGLE_SECRET:-fakefakefake}
 
-# Database settings.
 database:
-  driverClass: org.apache.derby.jdbc.EmbeddedDriver
-  url: jdbc:derby:/tmp/mydb;create=true
-  user: admin
-  password: admin
-  minSize: 8
-  maxSize: 200
-  validationQuery: values 1
-  properties:
-    hibernate.temp.use_jdbc_metadata_defaults: false
-    hibernate.dialect: org.hibernate.dialect.DerbyTenSevenDialect
-    hibernate.hbm2ddl.auto: update
-    hibernate.format_sql: true
+  driverClass: org.postgresql.Driver
+  url: ${JDBC_DATABASE_URL:-jdbc:postgresql://localhost:5432/gameswap_test?prepareThreshold=0&user=postgres}
 
```

Comment:
The other thing I think we need for heroku is a configurable username and password for postgres, but I'll check